aries_cloudagent/anoncreds/base.py

```diff
@@ -103,7 +103,10 @@ def supported_identifiers_regex(self) -> Pattern:
 
     async def supports(self, identifier: str) -> bool:
         """Determine whether this registry supports the given identifier."""
-        return bool(self.supported_identifiers_regex.match(identifier))
+        try:
+            return bool(self.supported_identifiers_regex.match(identifier))
+        except TypeError:
```

Comment:
Mostly curious, under what conditions were we receiving a TypeError?

Reply:
Looking at this again I'm not sure how I got this one to happen. I'll look it again and can probably remove it.